doc/BUILD.bazel

```diff
@@ -3,7 +3,7 @@ load("@bazel_skylib//rules:diff_test.bzl", "diff_test")
 load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
 
 _DOC_SRCS = [
-    "xcodeproj",
+    "xcodeproj.doc",
     "experimental",
```

Comment:
Is this ordered this way for a reason?

Reply:
I just thought `xcodeproj.doc` contains the "main" rules, but I can change it to alphabetical instead.

Reply:
Does it impact the generation in any way? If not then being alphabetical would be best.

Reply:
Doesn't seem like it. I changed it already in this PR and the diff was the same.